packages/flutter/lib/src/widgets/draggable_scrollable_sheet.dart

```diff
@@ -146,13 +148,19 @@ class DraggableScrollableSheet extends StatefulWidget {
   /// The default value is true.
   final bool expand;
 
+  /// A list of target points that the widget should snap to. [minChildSize] and
```

Comment:
nit: sentences in doc should follow proper grammar and start with a capital letter.

Reply:
The first paragraph of a doc comment should be a single sentence summary.

Reply:
What's the unit for snap points? Pixels? fractional values?